HW2/models.py

```diff
@@ -340,7 +342,7 @@ def forward_token(self, tokens, hidden):
       next_hidden = []
       x = self.embedding(tokens)
       for j, layer in enumerate(self.gru_cells):
-          x = layer(x, hidden[j])
+          x = self.dropout(layer(x, hidden[j]))
```

Comment:
x passer dans la connection recurrent. Il ne faut pas selon les instructions. Aussi il doit y avoir un dropout entre le embedding et les layers ainsi que entre les layers et le fully connected

Reply:
fixed in cbf15f863480a1636ee53bd91971bf8c2471d408